gslib/utils/text_util.py

```diff
@@ -35,6 +35,7 @@
 from gslib.lazy_wrapper import LazyWrapper
 from gslib.utils.constants import UTF8
 from gslib.utils.constants import WINDOWS_1252
+from gslib.tests.util import USING_JSON_API
```

Comment:
Please don't add test logic to non-test modules :)

Reply:
Whoops, that makes sense! Will remove the import and grab that value manually